app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if visitor.get("vaccine") is None:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+        elif visitor["vaccine"].get("expiration_date") < datetime.date.today():

```

Comment:
you not need 'elif' here couse func will end when raise error

Reply:
fix that